corehq/ex-submodules/couchforms/util.py

```diff
@@ -370,7 +370,7 @@ def process_xforms_for_cases(self, xform_lock_manager):
                     # todo: this property is only used by the MVPFormIndicatorPillow
                     instance.initial_processing_complete = True
 
-                    cases = case_db.get_cases_for_saving(now)
+                    cases = case_db.get_cases_for_saving(instance.received_on)
```

Comment:
I thought it would be good to have the same date for `form.received_on` and `case.modified_on`. What do you think?